UI/EmuScreen.cpp

```diff
@@ -923,20 +923,20 @@ void EmuScreen::render() {
 		screenManager()->getUIContext()->End();
 	}
 
-#ifdef USING_GLES2
 	// We have no use for backbuffer depth or stencil, so let tiled renderers discard them after tiling.
 	if (gl_extensions.GLES3 && glInvalidateFramebuffer != nullptr) {
```

Comment:
FWIW, I think some of our Linux builds relied on this ifdef due to glew.  Need a better loader...

-[Unknown]